### PR TITLE
meta: Add unreleased changes to changelog

### PR DESCRIPTION
**Pull request by jan-auer**
Adds unreleased changes to the changelog. For now, changes are grouped into the following sections:

- "Features": New or enhanced features relevant to users or the public behavior of Relay.
- "Bug Fixes": Fixes for the public behavior of Relay or that impact usage.
- "Internal": Both changes and bug fixes only relevant to the processing mode, trusted Relay operation.

Public behavior is mostly scoped to server features such as endpoints, compression, rate limits and caching behavior. Data scrubbing is explicitly included in this feature set. Explicitly excluded are store normalization and features only for trusted Relays.

This prepares for the `auto` changeset policy in craft: https://github.com/getsentry/craft/pull/93.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
 
+## Unreleased
+
+We have switched to [CalVer](https://calver.org/)! Relay's version is always in
+line with the latest version of [Sentry](https://github.com/getsentry/sentry).
+
+**Features**:
+
+- Proxy and managed Relays now apply clock drift correction based on the
+  `sent_at` header emitted by SDKs. (#581)
```
Comment (Swatinem) [inline CHANGELOG.md]: the issue links will not be automatically hyperlinked, you gotta do that manually 🤮

Comment (jan-auer) [reply CHANGELOG.md]: They will, see https://github.com/getsentry/symbolic/releases.

**Review by untitaker — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Swatinem) [reply CHANGELOG.md]: Not when you look at the changelog file:
https://github.com/getsentry/symbolic/blob/master/CHANGELOG.md

Comment (Swatinem) [reply CHANGELOG.md]: btw vscode has an official extension for this now, so it will hyperlink and show hover for those things in the editor.

Comment (jan-auer) [reply CHANGELOG.md]: > Not when you look at the changelog file

So far, we've been fine with taking that loss, but it's not a bad idea to improve on this. 👍

**Comment by jan-auer** [comment]
I've added links to the Changelog now. The problem with this will be that when craft copies the markdown source into the GitHub release. GitHub renders all line breaks literal in issues and releases, which will cause ugly line breaks:

image

---

We can either make craft reformat it, or we don't line-break in the Changelog. Preferences?

**Comment by untitaker** [comment]
I think it's fine not to line-break and I'd slightly prefer that over added complexity in craft

**Comment by jan-auer** [comment]
This is how it renders now:

https://github.com/getsentry/relay/blob/41be31cd6322358a471f0e9e0051076b79a52922/CHANGELOG.md

---

image

**Review by untitaker — APPROVED** [review]

**Pull request merged**